Jenkinsfile_servlet

```diff
@@ -28,44 +27,81 @@ node('linux'){
               globalMavenSettingsConfig: 'oss-settings.xml',
               mavenOpts: '-Xms2g -Xmx4g',
               mavenLocalRepo: ".repository") {
-        sh "mvn -V -B clean install -DskipTests -T10 -e"
+        sh "mvn -V -B -pl jetty-home -am clean install -DskipTests -T2 -e"
```

Comment:
This builds only `jetty-home` and its dependent modules.
We use it later, down below.
This took the build from 8 minutes to 1m40s